com.unity.shadergraph/Documentation~/TableOfContents.md

```diff
@@ -4,7 +4,7 @@
     * [My first Shader Graph](First-Shader-Graph) 
     * [Shader Graph Window](Shader-Graph-Window)  
         * [Blackboard](Blackboard)  
-        * [Master Preview](Master-Preview) 
+        * [Main Preview](Main-Preview) 
```

Comment:
Good catch! I took a look and saw that the title for `Main-Preview.md` (in the document itself) is Master Preview.

I'll ask the Shader Graph team what's the official name. Might need to do a mass find-and-replace in all the pages!

Reply:
replaced all instances i could find